api/controllers/UserController.js

```diff
@@ -49,6 +51,44 @@ module.exports = {
 
   },
 
+  sendInvite: function (req, res) {
+
+    User.findOne(req.param('id')).exec(function (err, user) {
+      if (err) return next(err);
```

Comment:
next() won't work in the controller context, we'll need to return a response here.